internal/testrand/rand.go

```diff
@@ -0,0 +1,89 @@
+// Copyright ©2020 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package testrand provides random generation and flags for testing.
+package testrand
+
+import (
+	"flag"
+
+	"golang.org/x/exp/rand"
+)
+
+var (
+	seedFlag    = flag.Uint64("testrand.seed", 1, "random seed for tests (0=randomize)")
+	extremeFlag = flag.Float64("testrand.extreme", 0, "probability of returning extreme values")
+	nanFlag     = flag.Float64("testrand.nan", 0, "probability of returning nan")
+)
+
+// TB is an interfaces that corresponds to *testing.T and *testing.B
+type TB interface {
+	Logf(format string, args ...interface{})
+}
+
+// Source corresponds to the interface in golang.org/x/exp/rand.Source.
+type Source interface {
```

Comment:
Should we just use `type Source = golang.org/x/exp/rand.Source`?

Reply:
Yeah, I was wondering whether to be explicit or not. I don't have a good argument for either approach.

Reply:
Here I think the type alias is a nice documentation of the reason for the type.

Reply:
Done